lua/telescope/builtin/files.lua

```diff
@@ -32,9 +33,26 @@ end
 files.live_grep = function(opts)
   local vimgrep_arguments = opts.vimgrep_arguments or conf.vimgrep_arguments
   local search_dirs = opts.search_dirs
-  opts.cwd = opts.cwd and vim.fn.expand(opts.cwd)
+  local grep_open_files = opts.grep_open_files
+  opts.cwd = opts.cwd and vim.fn.expand(opts.cwd) or vim.loop.cwd()
 
-  if search_dirs then
+  local filelist = {}
+
+  if grep_open_files then
+   local bufnrs = filter(function(b)
+      if 1 ~= vim.fn.buflisted(b) then return false end
+      return true
+    end, vim.api.nvim_list_bufs())
+    if not next(bufnrs) then return end
+
+    local tele_path = require'telescope.path'
+    for _, bufnr in ipairs(bufnrs) do
+      local file = vim.uri_to_fname(vim.uri_from_bufnr(bufnr))
+      table.insert(filelist, tele_path.make_relative(file, opts.cwd))
+    end
+  end
+
+  if search_dirs and not grep_open_files then
```

Comment:
..and I'm pretty sure this `not grep_open_files` isn't needed.
also you can do `vim.api.nvim_buf_get_name(bufnr)` instead of this URI stuff. 

I pressed approve to soon apparently :laughing: Could you fix this as well? :)

Reply:
Nice, I've updated it to use the nvim api.
Also removed the extra `not grep_open_files`.. I was kind of thinking we don't want to iterate the search dirs.

But a user shouldn't provide both options anyway.